### DISCUSSION TITLE
npm run dev NOT working

### DISCUSSION THREAD

**Discussion by shanewmurphy** [vercel/next.js#43202] [Help]
### Verify canary release

- [ ] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Don't understand why suddenly npm run dev isn't working 

npm create next-app

npm run dev --- above terminal window errors

![Screenshot 2022-11-21 at 20 34 35](https://user-images.githubusercontent.com/3694619/203153808-1370c12d-fb0a-4247-bb9a-139ef9dbe39b.png)

![Screenshot 2022-11-21 at 20 34 57](https://user-images.githubusercontent.com/3694619/203153838-f42a3a59-c86c-40b7-a811-8b50866d3692.png)

Anybody got a clue as to why I'd appreciate some help 

### What browser are you using? (if relevant)

Chrome 

### Describe the Bug

npm run dev not working 

### Expected Behavior

dev server localhost 

### Link to reproduction - Issues with a link to complete (but minimal) reproduction code will be addressed faster

No 

### To Reproduce

create next-app 
npm run dev

**Comment by shanewmurphy** [comment]
Ok my node version was out of date for next js

**Reply by viatanas** [reply]
This worked for me too. I uninstalled node and installed the latest version from https://nodejs.org. It worked for me after that.

**Comment by asb1512** [comment]
I had the same issue on Mac, and it turned out that the parent directory that the project was in had a '/' in it. All that I did, was change it to a '-', and the problem was resolved. I would assume npm, for some reason had trouble locating the directory with the slash in it because they're converted to colons by the OS.

**Comment by Volvein** [comment]
i executed npm run dev but project not running.

![Screenshot 2023-03-12 223123](https://user-images.githubusercontent.com/84660404/224560205-15b56f51-1e70-423b-8832-2fb5746a0d25.jpg)

**Reply by 96312** [reply]
What's your node version?

**Reply by Volvein** [reply]
Next.js 13 latest version bro

**Reply by KELOHL21** [reply]
I'm having the exact problem and i have no idea whats happening i tried starting some of my old projects and they start up perfectly fine

**Reply by KELOHL21** [reply]
with regards to versions Im using node version 18.15.0 Next.js 13 as well

**Reply by shanewmurphy** [reply]
Make sure ye are in the right folder in your terminal when running npm run dev

**Reply by jiseeeh** [reply]
I'm also having this problem (it immediately terminates), I thought the problem was moving my project from Linux to windows. I tried different things like making a fresh next app and reinstalling nodejs and yet this still happens.

**Reply by icyJoseph** [reply]
If you run: `node -p "process.arch"`, does the result match your architecture (32/64 bits)?

**Reply by ameer005** [reply]
I was facing the same problem because my node architecture was 32bits just install the 64bit version of node to solve this problem

**Reply by gnualmalki** [reply]
> If you run: `node -p "process.arch"`, does the result match your architecture (32/64 bits)?

output :

> arm64

**Reply by gnualmalki** [reply]
> I was facing the same problem because my node architecture was 32bits just install the 64bit version of node to solve this problem

How install it using terminal

**Reply by iFabruce** [reply]
> I was facing the same problem because my node architecture was 32bits just install the 64bit version of node to solve this problem

Thanks! Your answer make me save a precious time.
